.github/workflows/ci.yaml

```diff
@@ -1,45 +1,72 @@
 name: CI
-on: [push, pull_request]
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  schedule:
+    - cron: '0 0 * * FRI'
```

Comment:
Is there a reason for re-running CI every Friday?

Reply:
It is to let us know if a new rust version breaks CI or if any of the deps become outdated. It would be best to keep windows-sys crate version & the copied code in sync with the `home` crate. It would be fine to run this only twice or even once a month.

Reply:
Ah okay, that makes sense.